inc/blocks.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Manage Gutenberg-related logic.
```

Comment:
I wonder if the time has come to drop references to "Gutenberg"?

```suggestion
 * Manage block-related logic.
```